README.md

```diff
@@ -1,24 +1,203 @@
-# README
+# PENNYLANE
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 School Library ](#-school-library-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 PennyLane <a name="about-project"></a>
 
-* Configuration
+**PennyLane** is a web application designed to help users track and manage their financial transactions seamlessly. Whether it's monitoring your personal expenses or keeping a record of shared expenses within groups, PennyLane offers an intuitive and efficient platform to stay on top of your finances.
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+<summary>Back-End</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/installation/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
 
-* Services (job queues, cache servers, search engines, etc.)
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSLQ</a></li>
+    
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Front-End</summary>
+  <ul>
+    <li>HTML/CSS</li>
+    <li>JavaScript</li>
+    <li><a href="https://fontawesome.com/">Font Awesome</a></li>
+  </ul>
+</details>
+</details>
 
-* ...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **User Authentication:** PennyLane provides secure user authentication and registration processes, ensuring your financial data remains private.
+
+- **Expense Tracking:** Easily record and categorize your transactions to maintain a clear overview of your spending habits.
+
+- **User-Friendly Interface:** PennyLane's clean and intuitive interface ensures a smooth user experience, making financial management a breeze.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo & preview <a name="live-demo"></a>
+
+[live demo](https://rails-08wi.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Good job working on the project requirements. But it says I the project requirements. That you should add  your loom video link to your readme file.

Secondly you recorded a video quite alright but you did not show a piece of code  of a feature that you are most proud of in your project and also your video is too short. Please follow the following requirements for your video presentation:
Video presentation

Provide feedback about the video presentation in the code review summary.

The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, highlighting all the project features.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


It would be nice to record another video based on the above requirements please always pay attention and carefully read through project requirements.